tests/test_convertReferenceCatalog.py

```diff
@@ -55,11 +56,14 @@ def test_main_args(self):
             # Test with sets because the glob can come out in any order.
             self.assertEqual(set(run.call_args.args[0]), set(self.expected_files))
 
+        outdir.cleanup()
```

Comment:
Since you're using `TemporaryDirectory`, you don't need to call `cleanup`: it should be cleaned up automatically when it goes out of scope.

Reply:
You're right.  In some cases I see warnings, but I guess that's when it's not fully local?